iocore/net/PreWarmManager.h

```diff
@@ -136,7 +136,7 @@ enum class Stat {
   LAST_ENTRY,
 };
 
-using StatsIds          = std::array<int, static_cast<size_t>(PreWarm::Stat::LAST_ENTRY)>;
+using StatsIds          = std::array<ts::Metrics::IdType, static_cast<size_t>(PreWarm::Stat::LAST_ENTRY)>;
```

Comment:
Definitely not a big deal either way, but if you make this an array of `Metrics::IntType*`, you can use the static Metric::increment without needing the Metrix singleton instance.

Reply:
Yup, I thought it a bit, but I left it because it'll be big code change. 

Getting instance doesn't have noticeable over head, right?

Reply:
Yeh getting the instance should have no cost, the compiler should be smart about that. It’s a little bit of cost doing the indexing into the arrays

Reply:
But why would the code change be any bigger? You still just have to call the increment it write functions ?

Reply:
That pointer is the pointer directly to the atomic btw. It’s safe to use because the memory never moves or reallocates

Reply:
Each PreWarm Stat has 8 entries and getting metrics by the offset from the first entry. So changing it pointer makes... hmm...  let me try using `Metrics::IntType *`. My guess might be wrong.

https://github.com/apache/trafficserver/blob/84502fa3696c246bc8324e8ec05e556a26194ba6/iocore/net/PreWarmManager.h#L125-L137

Reply:
All right, I was just wrong. Using `Metrics::IntType *`, `ts::Metrics::increment` and ` `ts::Metrics::store` is much better. We don't need any sugar #10464. Thanks for checking.